controllers/likes.js

```diff
@@ -0,0 +1,198 @@
+import models from '../models/index';
+
+const { article, likes } = models;
+
+
+/**
+ * @description Likes and Favorites Controller class
+ */
+class Likes {
+  /**
+   * Adds two numbers together.
+   * @param {Object} req .
+   * @param {Object} res The User Object.
+   * @returns {Object}  Response object having message and status for liking artincle
+   */
+  static async likeArticle(req, res) {
+    try {
+      const { slug } = req.params;
+      const { user } = req;
+
+      const checkArticle = await article.findOne({ where: { slug, status: 'Published' } });
+      if (!checkArticle) {
+        res.status(404).send({
+          status: 404,
+          errorMessage: 'The Article you are trying to like is not found'
+        });
+      }
+
+      const arleadyLiked = await likes.findOne({
+        where: { userId: user.id, articleId: checkArticle.id }
+      });
+      if (arleadyLiked && (arleadyLiked.status === 'disliked' || arleadyLiked.status === null)) {
```

Comment:
Somehow this variable naming is misleading. `Now I wonder how alreadyLiked article can have a status equals to dislike` better if you worked on that in future.